### DISCUSSION TITLE
Is "Projects (classic) will be sunset on August 23, 2024" a problem?

### DISCUSSION THREAD

**Discussion by 4yman-0** [FreeTubeApp/FreeTube#5391] [General]
I saw that the repository uses both ["new" Github Projects](https://github.com/FreeTubeApp/FreeTube/projects) and [classic Github Projects](https://github.com/FreeTubeApp/FreeTube/projects?type=classic). Do we have to migrate now?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
We are aware of this and will migrate soon

**Reply by 4yman-0** [reply]
thank you
